src/state.rs

```diff
@@ -75,6 +75,7 @@ impl BarState {
 
         if let Reset::Elapsed | Reset::All = mode {
             self.state.started = now;
+            self.state.est.reset(now);
```

Comment:
This minimal change doesn't really make sense, since we're now unconditionally resetting the `Estimator` in all modes, so we can just pull it out of the first `if` and remove it from the second one?

Reply:
Thanks, I should have looked more closely here. I'll force push to the branch to fix this.